iphone/Classes/TiUITableView.m

```diff
@@ -2741,9 +2741,9 @@ - (void)presentSearchController:(UISearchController *)controller
       proxy = [proxy parent];
     }
     if ([proxy isKindOfClass:[TiWindowProxy class]]) {
-      searchControllerPresenter = [proxy windowHoldingController];
+      searchControllerPresenter = [[proxy windowHoldingController] retain];
```

Comment:
So it did need to retain? :-) Please also run the static analyser to ensure there are no static leaks.